cmd/run.go

```diff
@@ -37,12 +41,36 @@ type cmdRun struct {
 //
 //nolint:funlen,gocognit,gocyclo,cyclop
 func (c *cmdRun) run(cmd *cobra.Command, args []string) (err error) {
+	var logger logrus.FieldLogger = c.gs.logger
+	defer func() {
+		logger.Debugf("Everything has finished, exiting k6 with error '%s'!", err)
```

Comment:
>DEBU[0004] Everything has finished, exiting k6 with error '%!s(<nil>)'!

Do we want this output format? I think it could generate confusion. Maybe add the error only if this is not nil.

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/3c00d584bd1e87c8350d75fc168cfdbe6f9a3104..6f6b105ff041b2c317fa909d959bb77ea0946773)